src/js/components/Drop/DropContainer.js

```diff
@@ -15,6 +15,30 @@ import { StyledDrop } from './StyledDrop';
 import { OptionsContext } from '../../contexts/OptionsContext';
 import { useThemeValue } from '../../utils/useThemeValue';
 
+// Check if the element.tagName is an input, select or textarea
```

Comment:
This comment doesn't seem accurate to what the function is doing

Reply:
updated